src/pages/publication/publication_page/PublicationPageIdentifiers.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { FC, useEffect, useState } from 'react';
+import { Publication } from '../../../types/publication.types';
+import LabelContentRowForPublicationPage from './LabelContentRowForPublicationPage';
+import { ReferenceType } from '../../../types/references.types';
+import { useTranslation } from 'react-i18next';
+
+interface PublicationPageIdenfifiersProps {
+  publication: Publication;
+}
+
+const PublicationPageIdentifiers: FC<PublicationPageIdenfifiersProps> = ({ publication }) => {
+  const [isbn, setIsbn] = useState();
+  const { t } = useTranslation('publication');
+
+  useEffect(() => {
+    publication.reference.type === ReferenceType.BOOK && setIsbn(publication.reference?.book?.isbn);
+    publication.reference.type === ReferenceType.REPORT && setIsbn(publication.reference?.report?.isbn);
+  }, [publication]);
```

Comment:
syns det er litt rart å ha dette i en `useEffect` altså... kan vi ikke bare gjøre det direkte med to ternaries i return i stedet? da slipper vi ekstra rerender pga setState også